packages/design-system/src/themes/default/variables.css

```diff
@@ -24,34 +24,34 @@
 
   /* Spaces */
   --ads-spaces-0: 0;
-  --ads-spaces-1: 4;
-  --ads-spaces-2: 6;
-  --ads-spaces-3: 8;
-  --ads-spaces-4: 10;
-  --ads-spaces-5: 12;
-  --ads-spaces-6: 14;
-  --ads-spaces-7: 16;
-  --ads-spaces-8: 18;
-  --ads-spaces-9: 20;
-  --ads-spaces-10: 22;
-  --ads-spaces-11: 24;
-  --ads-spaces-12: 30;
-  --ads-spaces-13: 36;
-  --ads-spaces-14: 38;
-  --ads-spaces-15: 40;
-  --ads-spaces-16: 42;
-  --ads-spaces-17: 44;
+  --ads-spaces-1: 4px;
```

Comment:
In other places I've seen the px is given by the consumer of these variables + only the number is what is in ce as well. If we do this here then we'll have to change this everywhere it is being used, so I don't think this is a good idea